GenderPayGap.WebUI/Views/ManageOrganisations/OrganisationReportsTable.cshtml

```diff
@@ -52,19 +52,18 @@
                         <div>
                             @if (FeatureFlagHelper.IsFeatureEnabled(FeatureFlag.NewReportingJourney))
                             {
-                                <a class="govuk-button govuk-!-margin-top-4 govuk-!-margin-bottom-0" 
-                                   href="@Url.Action("ReportOverview", "ReportOverview",
-                                             new {encryptedOrganisationId = encryptedOrganisationId, reportingYear = detailsForYear.ReportingYear})">
+                                <a class="govuk-button govuk-!-margin-top-4 govuk-!-margin-bottom-0 create-report-@detailsForYear.ReportingYear"
+                                   href="@Url.Action("ReportOverview", "ReportOverview", new {encryptedOrganisationId = encryptedOrganisationId, reportingYear = detailsForYear.ReportingYear})">
                                     @detailsForYear.GetReportButtonText()
                                 </a>
                             }
                             else
                             {
                                 var encryptedRequestParam = Encryption.EncryptAsParams(Model.Organisation.OrganisationId.ToString(), detailsForYear.ReportingYear.ToString(), detailsForYear.DoesReturnOrDraftReturnExistForYear().ToString());
-                                
+
                                 <a role="button"
                                    href="@Url.Action("ReportForOrganisation", "Organisation", new {request = encryptedRequestParam})"
-                                   class="govuk-button govuk-!-margin-top-4 govuk-!-margin-bottom-0">
+                                   class="govuk-button govuk-!-margin-top-4 govuk-!-margin-bottom-0 create-report-@detailsForYear.ReportingYear">

```

Comment:
This fills more the role of an `id` than a `class` so might make more sense there? Alternatively - on e.g. a Selenium test you'd put in a custom attribute to clearly differentiate it from functional attributes:

```
<a class="govuk-button" data-id-for-tests="create-report-@detailsForYear.ReportingYear">some text here</a>
```
and then you ought to be able to do
```
css("a[data-id-for-tests='create-report-2020']", "href").find()
```